src/main/scala/viper/silver/plugin/standard/termination/transformation/ExpTransformer.scala

```diff
@@ -75,6 +75,29 @@ trait ExpTransformer extends ErrorReporter {
           Seqn(Seq(right), Nil)()
       }
       Seqn(Seq(left, rightSCE), Nil)()
+    case (ase: AnySetExp, c) => ase match {
+      case AnySetCardinality(s) => Seqn(Seq(transformExp(s, c)), Nil)(ase.pos)
+      case AnySetUnion(l, r) => Seqn(Seq(transformExp(l, c), transformExp(r, c)), Nil)(ase.pos)
+      case AnySetIntersection(l, r) => Seqn(Seq(transformExp(l, c), transformExp(r, c)), Nil)(ase.pos)
+      case AnySetSubset(l ,r) => Seqn(Seq(transformExp(l, c), transformExp(r, c)), Nil)(ase.pos)
+      case AnySetMinus(l, r) => Seqn(Seq(transformExp(l, c), transformExp(r, c)), Nil)(ase.pos)
+      case AnySetContains(elem, s) => Seqn(Seq(transformExp(elem, c), transformExp(s, c)), Nil)(ase.pos)
+      case MapDomain(base) => Seqn(Seq(transformExp(base, c)), Nil)(ase.pos)
+      case MapRange(base) => Seqn(Seq(transformExp(base, c)), Nil)(ase.pos)
+      case unsupportedExp => transformUnknownExp(unsupportedExp, c)
```

Comment:
I'm confused by "unsupportedExp". Is this case actually possible, are there cases we we didn't already cover?
If it's actually unsupported, we should probably raise an exception (or just not add this case so we get a MatchError)? But if there are cases where we intentionally take this branch and perform the default conversion, then we should probably call this "otherExp" or something like that.

Reply:
The only client of this trait used to report an error by overwriting `unsupportedExp`. I'm now handling all (built-in) Viper expressions and reporting an error for expressions contributed by plugins